kernel/src/main/scala/cats/kernel/Order.scala

```diff
@@ -139,7 +139,16 @@ abstract class OrderFunctions[O[T] <: Order[T]] extends PartialOrderFunctions[O]
     ev.max(x, y)
 }
 
-object Order extends OrderFunctions[Order] {
+trait OrderToOrderingConversion {
+  /**
+   * Implicitly derive a `scala.math.Ordering[A]` from a `Order[A]`
+   * instance.
+   */
+  implicit final def catsKernelOrderingForOrder[A](implicit ev: Order[A]): Ordering[A] =
+    ev.toOrdering
+}
+
+object Order extends OrderFunctions[Order] with OrderToOrderingConversion {
```

Comment:
Is there any benefit to having `implicit final def catsKernelOrderingForOrder[A](implicit ev: Order[A]): Ordering[A]` inside the `Order` companion object? I don't think that scala will look here when it is searching for an `Ordering` instance. I could see a minor benefit of having a non-`implicit` version of this method inside the companion object. As it stands it seems like it's just a liability as I would expect you to get ambiguous implicits if you've imported both `cats.implicits._` and `cats.kernel.Order._`.